### ISSUE TITLE
Manager.is_locked() does not use expiration

### ISSUE THREAD

**Issue by thenewguy** [vikingco/django-locking#5]
Shouldn't Manager.is_locked() additionally check if the existing locks have already expired?

In other words https://github.com/vikingco/django-locking/blob/master/locking/models.py#L71 should read something like the following:
```
return qs and any([not lock.is_expired for lock in qs])
```

**Comment by thenewguy** [comment]
Or actually just:
```
return any([not lock.is_expired for lock in qs])
```

Sorry for untested code.  Just happened across this project seeing if something like it already existed

**Comment by thenewguy** [comment]
This could be done in sql as well.  It would be a simpler query if there was an expires_at field on the model to query against.  That would be a design change though because of the created_on + max_age route already taken.

**Comment by nielsvanoch** [comment]
Fixed in 1.1.0.
